app/routes/index.js

```diff
@@ -19,23 +18,7 @@ router.get('/', function(request, response, next) {
 
 router.get('/home', (request, response) => {
     if (request.user) {
-        Influencer.findOne({_id: request.user.id})
```

Comment:
We were fetching the user (influencer) again, but passport already fetches it first in middleware and adds it to `request.user`.